config.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  expose-external:
+    description: |
+      String to determine how to expose the MySQLRouter externally from the Kubernetes cluster.
+      Possible values: 'false', 'nodeport', 'loadbalancer'
```

Comment:
@paulomach any ideas for alternatives? `expose-external: none`? `expose-external: clusterip`?

Reply:
additionally, the possible values are `false` and `nodeport` in [kafka-k8s](https://github.com/canonical/kafka-k8s-operator/blob/main/config.yaml#L112). it may not be a good idea to introduce inconsistencies

Reply:
oh bummer, ok maybe for another time.

Reply:
Interestingly, it seems it was a Marc nitpick also that Mykola did not responded to in the original [kafka spec](https://docs.google.com/document/d/1JNrThrEULwA2nxi5ZQ9-mkbEGi5SS6N7Ku7MIyfkWbo/edit?disco=AAABUX0Z-sU)

Reply:
> default false may imply true is valid value. Change to something else? no none

👋🏻 I subscribe Paulo's comment here.

I understand we are "compromising" in order to have a similar interface across DP charms (i.e. [kafka-k8s](https://github.com/canonical/kafka-k8s-operator/blob/f4d9336d96fc7e4aa47a5afb112b5dbf8d5ade6e/config.yaml#L115-L118)), but it is confusing and we all should make an effort to reduce these occurrences in the future.